app/points/domain/workload-owner.js

```diff
@@ -0,0 +1,20 @@
+const assertNumber = require('./validation/assert-number')
+
+class WorkloadOwner {
+  constructor (id, offenderManagerId, workingHoursId, teamId) {
+    this.id = id
+    this.offenderManagerId = offenderManagerId
+    this.workingHoursId = workingHoursId
+    this.teamId = teamId
+    this.isValid()
+  }
+
+  isValid () {
+    assertNumber(this.id, 'id')
+    assertNumber(this.offenderManagerId, 'offenderManagerId')
+    assertNumber(this.workingHoursId, 'workingHoursId')
```

Comment:
Working hours is nullable, because if they have the default hours we just inherit it from the global conf.